projects/plugins/boost/app/assets/src/js/lib/stores/minify.ts

```diff
@@ -28,3 +28,9 @@ export const useMetaQuery = ( key: MinifyMetaKeys, onSuccess?: ( newState: strin
 
 	return [ data || [], updateValues ] as const;
 };
+
+export const useShowMinifyLegacy = () => {
+	const [ query ] = useDataSync( 'jetpack_boost_ds', 'minify_legacy_notice', z.boolean() );
+
+	return query;
```

Comment:
It's a nitpick but since we aren't using other properties, can we narrow it down to data? This will make it cleaner on the end it is being used.

```suggestion
	const [ query: { data } ] = useDataSync( 'jetpack_boost_ds', 'minify_legacy_notice', z.boolean() );

	return data;
```

Reply:
We're using the `refetch` function within the Minify components so the query object is needed unfortunately, e.g.
https://github.com/Automattic/jetpack/blob/866d7dd70efba716d1ae55bb5f23793f7dbc2cd6/projects/plugins/boost/app/assets/src/js/features/minify-js/minify-js.tsx#L22